pkg/api/unversioned/group_version.go

```diff
@@ -29,8 +29,8 @@ import (
 // `resource.group.com` -> `group=com, version=group, resource=resource` and `group=group.com, resource=resource`
 func ParseResourceArg(arg string) (*GroupVersionResource, GroupResource) {
 	var gvr *GroupVersionResource
-	s := strings.SplitN(arg, ".", 3)
-	if len(s) == 3 {
+	if strings.Count(arg, ".") == 3 {
+		s := strings.SplitN(arg, ".", 3)
```

Comment:
I *think* so?  I've been seeing bizarre indentation in github, but
gofmt continues to be a no-op for me.